oeps/oep-1.rst

```diff
@@ -0,0 +1,266 @@
+  address the problem that the OEP solves, or why Open edX would be significantly
+  improved by adopting the best practice. OEP submissions without sufficient
+  motivation may be rejected outright.
+
+*Rationale*
+  The rationale fleshes out the specification by describing what motivated the
+  design and why particular design decisions were made. It should describe
+  alternate designs that were considered and related work, e.g. how the
+  feature is supported in other systems.
+
+  The rationale should provide evidence of consensus within the community
+  and discuss important objections or concerns raised during discussion.
+
+*Backwards Compatibility*
+  All OEPs that introduce backwards incompatibilities must include a section
+  describing these incompatibilities and their severity. The OEP must explain
+  how the author proposes to deal with these incompatibilities. OEP submissions
+  without a sufficient backwards compatibility treatise may be rejected outright.
+
+*Reference Implementation*
+  The reference implementation must be completed before any OEP is given status
+  "Final", but it need not be completed before the OEP is accepted. While there
+  is merit to the approach of reaching consensus on the specification and rationale
+  before writing code, the principle of "rough consensus and running code" is still
+  useful when it comes to resolving many discussions of API details.
+
+  The final implementation must include test code and documentation, following the
+  `Open edX Contribution Guidelines`_.
+
+.. _Open edX Contribution Guidelines: http://edx.readthedocs.org/projects/edx-developer-guide/en/latest/process/index.html
+
+OEP Formats and Templates
+-------------------------
+
+OEPs are UTF-8 encoded text files using the `reStructuredText`_ format.
+ReStructuredText [8] allows for rich markup that is still quite easy to read,
+but also results in good-looking and functional HTML. OEPs are rendered to HTML
+using Sphinx. An OEP template can be found `in the repo`_.
+
+.. _reStructuredText: http://docutils.sourceforge.net/rst.html
+.. _in the repo: https://github.com/cpennington/open-edx-proposals/blob/master/oep-template.rst
+
+OEP Header Preamble
+-------------------
+Each OEP must begin with an ReST tabel with meta-data about the OEP. The rows must
+appear in the following order. Rows in italics are optional and
+are described below. All other rows are required.
+
+
++---------------+-------------------------------------------+
+| OEP           | <link to OEP>                             |
++---------------+-------------------------------------------+
+| Title         | <OEP title>                               |
++---------------+-------------------------------------------+
+| Last-Modified | <date string>                             |
++---------------+-------------------------------------------+
+| Author        | <list of authors' real names and          |
+|               | optionally, email addrs>                  |
```

Comment:
Spell out "addresses", and why are they optional? It's not too much to ask that we can reach the author.